frontend/src/components/activity/content/Storyboard.vue

```diff
@@ -1,124 +1,65 @@
 <template>
-  <card-content-node v-bind="$props">
-    <v-container fluid>
-      <v-row no-gutters class="text-subtitle-2">
-        <v-col cols="2">
-          {{ $tc('contentNode.storyboard.entity.section.fields.column1') }}
-        </v-col>
-        <v-col cols="7">
-          {{ $tc('contentNode.storyboard.entity.section.fields.column2Html') }}
-        </v-col>
-        <v-col cols="2">
-          {{ $tc('contentNode.storyboard.entity.section.fields.column3') }}
-        </v-col>
-        <v-col cols="1" />
-      </v-row>
-
-      <api-sortable
-        v-slot="sortable"
+  <card-content-node v-resizeobserver.debounce="onResize" v-bind="$props">
+    <component :is="variant === 'default' ? 'table' : 'div'">
+      <thead v-if="variant === 'default'">
+        <tr>
```

Comment:
We really want this to be a normal table and not v-row/v-col components as before?

Reply:
Pretty sure

Reply:
I'm unfortunately not really knowledgeable an a11y, so cannot really judge on this. What seems strange to me is that we use table elements for something that doesn't really look like a table (in dense variant) and where vuetify provided the v-row/v-col for exactly this purpose. 

Also in default variant there's an extra div in the table:
![image](https://user-images.githubusercontent.com/449555/223143627-f54cc715-9ab4-4f63-9d3c-238565674da7.png)

And in dense variant there's even no table element:
![image](https://user-images.githubusercontent.com/449555/223143832-66208baa-495c-4107-b678-74837a5348ee.png)

As far as I know, both are not valid HTML.

Reply:
This should be sorted now